docs/source/overview.rst

```diff
@@ -1,12 +1,12 @@
 Overview
 ========
 
-django-declarative-apis is a framework built on top of Django aimed at teams implementing RESTful APis. It provides
+django-declarative-apis is a framework built on top of Django aimed at teams implementing RESTful APIs. It provides
 a simple interface to define endpoints declaratively. Some benefits to using django-declarative-apis:
 
 * Define endpoints declaratively
```

Comment:
I /think/ you want `-` here rather than `*`. Looking at the rendered output it doesn't show up as a list.

Reply:
I used to have that issue as well, but @bgrant figured out that it was happening because of `sphinx_rtd_theme` version. Upgrading `sphinx_rtd_theme` fixed it for me. So I can change the version to a later one in the requirements.txt.

Reply:
I just realized that I had actually upgraded `sphinx_rtd_theme` in this PR. It was updated from 0.4.2 to 0.5.2 ( I got the version from [here](https://pypi.org/project/sphinx-rtd-theme/#history)). 
Now I am not sure what else could be the reason for the list not rendering correctly. 🤔